sdcm/tester.py

```diff
@@ -793,6 +793,10 @@ def init_resources(self, loader_info=None, db_info=None,
         if cluster_backend == 'aws' or cluster_backend == 'aws-siren':
             self.get_cluster_aws(loader_info=loader_info, db_info=db_info,
                                  monitor_info=monitor_info)
+
+            if cluster_backend == 'aws-siren':
```

Comment:
I don't think that would be a good idea, cause it won't create the loaders...
I think adding this condition in the same place you check `if seeds_selector == 'reflector' or Setup.REUSE_CLUSTER:`

Reply:
In case of not REUSE, RF of system_auth will be changed. Do we need it for cloud?
```
        system_auth_rf = self.params.get('system_auth_rf', default=3)
        if system_auth_rf and not cluster.Setup.REUSE_CLUSTER:
            self.log.info('change RF of system_auth to %s', system_auth_rf)
            node = self.db_cluster.nodes[0]
            with self.cql_connection_patient(node) as session:
                # pylint: disable=no-member
                session.execute("ALTER KEYSPACE system_auth WITH replication = "
                                "{'class': 'org.apache.cassandra.locator.SimpleStrategy', "
                                "'replication_factor': %s};" % system_auth_rf)
            self.log.info('repair system_auth keyspace ...')
            node.run_nodetool(sub_cmd="repair", args="-- system_auth")```

Reply:
Why the loaders won't be created? 
I run the test now. All run OK

Reply:
Yes, we should alter the system RF in cloud.

Reuse flag imply not creating resources, maybe when you change it it already after that logic was applied. I still rather not change it's value.

Reuse means an already configured by SCT cluster. Cloud is new a reuse, but a slightly different case. let's try not to mix them.

Reply:
Changed. Please, review